update_copyright.d

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/env rdmd
+
+/**
+Update the copyright notices in source files so that they have the form:
+---
+    Copyright XXXX-YYYY by The D Language Foundation, All Rights Reserved
+---
+It does not change copyright notices of authors that are known to have made
+changes under a proprietary license.
+
+Copyright:  Copyright (C) 2017-2018 by The D Language Foundation, All Rights Reserved
+
+License:    $(WEB boost.org/LICENSE_1_0.txt, Boost License 1.0)
+
+Authors:    Iain Buclaw
+
+Example usage:
+---
+update-copyright.py --update-year src/dmd
+---
+*/
+
+module tools.update_copyright;
```

Comment:
maybe just `update_copyright`?  When the repo is downloaded to a local machine, the tools repo may be named something else.

Reply:
Top-level module names aren't particularly encouraged.

Reply:
How come?  I don't see any benefit to qualifying the module name with a directory that can change, I only see the potential for breakage.

Reply:
Breakage of what? How are _you_ using this standalone module?

Reply:
Huh?

Reply:
I see you've edited away your original comment.

Anyway, having a module name means that you get predictable identifiers in the output binary.